src/backend/joanie/core/models/products.py

```diff
@@ -1035,6 +1059,67 @@ def get_equivalent_course_run_dates(self):
             for key, value in aggregate.items()
         }
 
+    def _get_schedule_dates(self):
+        """
+        Return the schedule date for the order.
+        """
+        try:
+            start_date = self.contract.student_signed_on
+        except Order.contract.RelatedObjectDoesNotExist as e:
+            logger.error(
+                "Contract does not exist, cannot retrieve retraction date",
```

Comment:
```suggestion
                "Contract does not exist, cannot retrieve withdrawal date",
```

I think in a financial context, withdrawal is more appropriated